# app name
string name
# user-friendly display name of application
string display_name
# the version of the package currently installed
string version
# latest version of the package avaliable
string latest_version
# the detailed description of the app
string description
# icon for showing app
Icon icon
# hidden apps are not show - used for cases where multiple apps are in a deb
bool hidden